src/client/components/Button/Button.styles.css

```diff
@@ -0,0 +1,13 @@
+.button-register-yourself {
```

Comment:
Rename to `.button`.

Reply:
renamed to button1. If you use button styling changes.